stubs/inertia/resources/js/Components/Modal.vue

```diff
@@ -17,18 +17,27 @@ const props = defineProps({
 });
 
 const emit = defineEmits(['close']);
+const dialog = ref();
+const showSlot = ref(props.show);
 
 watch(() => props.show, () => {
     if (props.show) {
         document.body.style.overflow = 'hidden';
+        showSlot.value = true;
+        dialog.value?.showModal();
     } else {
         document.body.style.overflow = null;
+        setTimeout(() => {
+            dialog.value?.close();
+            showSlot.value = false;
+        }, 200);
     }
 });
 
 const close = () => {
     if (props.closeable) {
         emit('close');
+        e.preventDefault();
```

Comment:
I didn't test on a fresh installation. I just tried to update my already existing component and got this:

`Uncaught ReferenceError: e is not defined`

Reply:
I don't think it's really necessary at all. I've removed it and sumitted a new PR. https://github.com/laravel/jetstream/pull/1444